ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Operating system version & Platform:
+IPFS commit version (from `ipfs version --commit`):
+IPFS repo version (from `ipfs version --repo`):
```

Comment:
I don't think we need the repo version, if we know the commit, or even just the general semver we can infer it. It might be better to ask for 'ipfs version (and commit, if known)'